beslab_genesis.yaml

```diff
@@ -1,54 +1,356 @@
+#           - GDB
+#           - Makefile
+#           - C
+#           - Python
+#           - Perl
+#           - SQL
+#           - MYSQL
+#           - PGSQL
+#           - APACHE
+#           - npm
+#           - maven
+#           - pip3
+#           - vscode
+#           - gradle
+#           - docker
+#           - 
+#           etc
+###########################################################
+# BESLAB_DEV_TOOLS:
+#    - JAVA
+#    - Python
+#    - APACHE
+#    - MAKEFILE
+#    - GDB
+#    - maven
+#    - docker
+#    - pip3
+
+## IF JAVA is to be installed.
+###########################################################
 BESLAB_JAVA_FLAVOUR: openjdk
 BESLAB_JAVA_VERSION: 8
-# BESLAB_JAVA_HOME: 
-BESLAB_PYTHON_VERSION: 3.10
-BESLAB_SBOM: spdx-sbom-generator
-BESLAB_SBOM_VERSION: "v0.0.15"
-BESLAB_LICENSE_COMPLIANCE: fossology
-BESLAB_SAST: sonarqube
-BESLAB_SAST_VERSION: "sonarqube-9.8.0.63668"
-BESLAB_BES_TOOL:
-  - besman
-  - bes-dev-kit
-BESLAB_PACKAGE_MANAGER:
-  - npm
-  - gradle
-  - maven
-  - pip
-# Change the below to private if you have a private package manger repository.
-BESLAB_ARTIFACT_PATH: /etc
-BESLAB_PACKAGE_MANAGER_SCOPE: public
-BESLAB_ARTIFACT_REPO_SCOPE: public
-BESLAB_PRIVATE_ARTIFACT_REPOS:
-  - jfrog
-BESLAN_PUBLIC_ARTIFACT_REPO:
-  - maven
-BESLAB_IMAGE_REPOS:
-  - docker
+BESLAB_JAVA_HOME: /opt/tools/
+
+## IF Python is to be installed.
+###########################################################
+#BESLAB_PYTHON_VERSION: 3.10
+
+## IF maven to be installed
+###########################################################
+#BESLAB_MAVEN_VERSION: 
+
+## IF Gradle to be installed.
+###########################################################
+#BESLAB_GRADLE_VERSION:
+
+#[Note]: ADD the dev tool version in the format BESLAB_<tool name>_VERSION.
+#
+
+###########################################################
+# BESLAB_SBOM: SBOM generator tool to be installed.
+#    If commented no SBOM tool will be installed.
+###########################################################
+#BESLAB_SBOM: spdx-sbom-generator
+#BESLAB_SBOM_VERSION: "v0.0.15"
+
+# BESLAB_LICENSE_COMPLIANCE: tool for licensing.
+#    If commented no Licensing tool will be installed.
+###########################################################
+#BESLAB_LICENSE_COMPLIANCE: fossology
+
+# BESLAB_SAST: Static code analysis tool to be installed.
+#    If commented no SAST tool will be installed.
+###########################################################
+#BESLAB_SAST: sonarqube
+#BESLAB_SAST_VERSION: "sonarqube-9.8.0.63668"
+
+
+# BESLAB_BES_TOOL: List of tools to be installed from 
+#    Be-Secure community.
+#BESLAB_BES_TOOL:
+#  - besman
+#  - bes-dev-kit
+#  - BesLighthouse
+
+## Applicable only if BeSLighthouse is to be installed.
+###########################################################
+
+# BESLAB_BESLIGHTHOUSE_RELEASE_VERSION: 0.16.2
+# BESLAB_BESLIGHTHOUSE_RELEASE_URL: https://github.com/Be-Secure/BeSlighthouse/archive/refs/tags/0.16.2.tar.gz
```

Comment:
Should be uncommented, otherwise the code wont work. Instead of beslighthouse, should be dashboard